src/components/panels/SentientOutpostsPanel.vue

```diff
@@ -29,17 +34,23 @@ import TimeBadge from '@/components/TimeBadge.vue';
 import HubPanelWrap from '@/components/HubPanelWrap';
 import HubImg from '@/components/HubImg.vue';
 
-import moment from 'moment';
+//import moment from 'moment';
 
 import sentient from '@/assets/img/factions/sentient.svg';
 
 export default {
   name: 'SentientOutpostsPanel',
   props: ['sentientOutposts'],
+  watch: {
+    mission: function () {
+      this.active = new Date().getTime();
```

Comment:
uhhh, let's just use outpost.activation? i can go fix it on the api, this is just kinda jank 🤔